base/_base.scss

```diff
@@ -0,0 +1,28 @@
+@charset "UTF-8";
```

Comment:
I think this file name should be reserved for the "roll up" partial include file. I propose renaming this file to _box-model.scss and creating a base.scss for all of the imports.